extensions/spark/kyuubi-spark-authz/src/main/scala/org/apache/kyuubi/plugin/spark/authz/util/AuthZUtils.scala

```diff
@@ -87,6 +87,12 @@ private[authz] object AuthZUtils {
   lazy val isSparkV31OrGreater: Boolean = SPARK_RUNTIME_VERSION >= "3.1"
   lazy val isSparkV32OrGreater: Boolean = SPARK_RUNTIME_VERSION >= "3.2"
   lazy val isSparkV33OrGreater: Boolean = SPARK_RUNTIME_VERSION >= "3.3"
+  lazy val isSparkV34OrGreater: Boolean = SPARK_RUNTIME_VERSION >= "3.4"
+  lazy val isSparkV35OrGreater: Boolean = SPARK_RUNTIME_VERSION >= "3.5"
+
+  lazy val SCALA_RUNTIME_VERSION: SemanticVersion =
+    SemanticVersion(scala.util.Properties.versionNumberString)
+  lazy val isScalaV213OrGreater: Boolean = SCALA_RUNTIME_VERSION >= "2.13"
```

Comment:
simply define `isScalaV213`, Scala 3 is a major change and is quite far from the current stage.

Reply:
Done